src/main/java/seedu/address/model/module/TutorialTeam.java

```diff
@@ -0,0 +1,190 @@
+package seedu.address.model.module;
+
+import static seedu.address.commons.util.AppUtil.checkArgument;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.ArrayList;
+
+import seedu.address.model.person.Person;
+
+/**
+ * Represents a Module's tutorial team.
+ * Guarantees: immutable; is valid as declared in
+ * {@link #isValidTeamName(String)}
+ */
+public class TutorialTeam {
+
+    public static final String MESSAGE_CONSTRAINTS = "Please enter a valid tutorial team name "
+            + "eg. Team 1, and it should not be blank";
+    public static final String MESSAGE_NAME_CONSTRAINTS = "Team names should only contain alphanumeric "
+            + "characters and spaces, and it should not be blank";
+    public static final String MESSAGE_SIZE_CONSTRAINTS = "Team size should be a positive integer";
+
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
```

Comment:
Maybe you show limit the length of the team name? this way we avoid overflow issues on the UI possibly.

Reply:
We can do this. In this case, I followed the student naming regex for this.

Reply:
We can discuss more once we have finalised the UI as well as the constraints